### ISSUE TITLE
Seeing a bit of the original bar on the right when resizing

### ISSUE THREAD

**Issue by AnthonySLWhite** [Grsmto/simplebar#303]
## 🐛 Bugs

Using flex on an element with wrap (not sure if without the bug occurs) and then resizing window or zooming makes the hidden scrollbar appear a bit on the right.
The problem is the element with the class "simplebar-offset" is setting itself with a position of `right: -11px` problem is, it isn't enough. It has to be `right: -12px` sometimes.

## Example
I'm sorry this was a bit in a hurry. I just thought it was good to point out.
Unfortunately I can't rebuild the issue for demonstration at the moment.

But I have the following example:
```
<div class="flex-row">
	<div data-simplebar class="scroll wrapper">
		<section>
			<h1>Title</h1>
			<div class="flex-row-wrap">
				...elements from array
			</div>
		</section>
	</div>
</div>
```
The inner flex element makes the native scrollbar appear 1 pixel from the right when the window is resized.
The issue only happens at some breakpoints

## Quick Fix
I fixed the bug temporarily using:
```
.simplebar-offset {
  right: -1.1rem !important;
}
```
And until now it worked responsively but a pixel off sometimes.

## Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| SimpleBar        |   ^3.1.5
| Browser          |Chrome & Vivaldi (Version > 73.0.3683.103)
| npm         	   |   6.9.0
| Operating System | Windows 10

**Comment by Grsmto** [comment]
Hi,
Thanks for your report. Unfortunately if you don't follow the issue template that won't help at all. Especially the `Your environment` part.

**Comment by AnthonySLWhite** [comment]
Hi @Grsmto I've updated the issue. Sorry but I didn't have the time to elaborate. Just thought I should share with you

**Comment by Grsmto** [comment]
Should be fixed in 4.1.0. Please try and re-open if needed, thanks!
